src/main/java/be/acara/events/service/EventServiceImpl.java

```diff
@@ -162,4 +163,11 @@ public Event editEvent(Long id, Event event) {
         }
         return eventRepository.findAll(specification, pageable);
     }
+
+    @Override
+    public void addAttendee(Event event, User user) {
+        event.getAttendees().add(user);
+        user.getEvents().add(event);
+        eventRepository.save(event);
```

Comment:
the .save(event) can be removed. This method could actually be moved to the Event domain class.